lib/userpoint.php

```diff
@@ -192,15 +195,14 @@ function elggx_userpoints_get($user_guid) {
 		'pending' => 0,
 	];
 
-	$entities = elgg_get_entities_from_metadata([
+	$entities = new ElggBatch('elgg_get_entities_from_metadata', [
```

Comment:
why did you change this, the result is exactly the same

Reply:
I thought it would be better to define the ElggBatches in the same way throughout the whole plugin. As the "traditional" approach was still used more often than the new one I was just lazy and changed the code back at the two places the new way was used.

Also, I wasn't 100% sure about ```setIncrementOffset()``` usage, i.e. if it's set to false automatically if the 'batch' option is set to true or if I would have to set it to false as before (it's necessary in the second location when deleting entities is done in the loop but you had removed the line before the loop). I guess I just could have asked to make sure that my feeling is right that it works in the same way as before and I would have to set it to false explicitely.

Actually, I hadn't known about the new 'batch' option for the e_get*() functions and I like it. As the PR is not ready anyway I can switch all ElggBatches definitions to the new approach.

Reply:
Found the 'batch_inc_offset' option is available. That answers my question.